.readthedocs.yml

```diff
@@ -6,7 +6,6 @@ build:
     python: mambaforge-4.10
   jobs:
     post_checkout:
-      - git fetch --unshallow
```

Comment:
@bjlittle and I now understand that `git fetch --unshallow` is invalid for cf-units, because cf-units (unlike Iris) is NOT a shallow repository. This might change in future if anything 'nasty' happens in the Git history, but the aim/hope is that it remains unshallow.

Please could we have an explanatory comment here, in case things change or this config is used as a template for another repo.

Reply:
@trexfeathers Done :+1:

See 61d3280.

Reply:
We could auto-detect `.git/shallow`, but perhaps that is a tad over-engineering here.